app/models/signature.rb

```diff
@@ -61,7 +61,7 @@ def to_csv_line
   end
 
   def state_symbol
-    us_states_hash[state]
```

Comment:
Nice, looks like this model can stop including `ApplicationHelper` now. :confetti_ball:

Reply:
Oo great!